deployments/examples/ocis_full/ocis.yml

```diff
@@ -16,8 +16,6 @@ services:
     # therefore we ignore the error and then start the ocis server
     command: ["-c", "ocis init || true; ocis server"]
     environment:
-      # enable the notifications service as it is not started automatically
```

Comment:
Please do not delete that, we have the normal use case with an email server without inbucket too.

Reply:
done